downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -95,11 +95,10 @@ sso_keystore_password=''
 
 # Boolean flag used to verify Automation Controller's
 # web certificates when making calls from Automation EDA Controller.
-#
 # automationedacontroller_controller_verify_ssl = true
-
-# Certificate and key to install in Automation Hub node
-# automationhub_ssl_cert=/path/to/automationhub.cert
-# automationhub_ssl_key=/path/to/automationhub.key
+#
+# Certificate and key to install in Automation EDA Controller node
```

Comment:
Would it be worth adding the attribute here for the EDA product name consistency?

{EDAcontroller}